src/snowflake/connector/connection.py

```diff
@@ -204,6 +204,10 @@ def DefaultConverterClass() -> type:
         True,
         bool,
     ),  # Whether to log imported packages in telemetry
+    "json_result_force_utf8_decoding": (
+        False,
+        bool,
+    ),  # Whether to force the JSON content to be decoded in utf-8, it only works when result format is JSON
```

Comment:
Let's not say "only works", what about "is only effective"?

Reply:
updated